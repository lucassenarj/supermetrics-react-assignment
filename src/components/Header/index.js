import React from "react";
import { Link } from "react-router-dom";
import Logo from "./../../assets/images/logo.png";
import Menu from "./../Menu";
import "./index.scss";

function Header() {
  return (
    <header className="header">
      <Link to="/dashboard">
        <img src={Logo} alt="Home" />
      </Link>

      <Menu />
    </header>
  );
};

export default Header;
