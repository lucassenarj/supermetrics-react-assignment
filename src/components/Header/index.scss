@import "./../../assets/colors.scss";

.header {
  background: $background;
  display: flex;
  flex-wrap: wrap;
  justify-content: space-between;
  align-items: center;
  padding: 20px;
}

.header img {
  height: 60px;
}