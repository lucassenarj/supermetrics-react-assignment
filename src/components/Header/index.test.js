import React from "react";
import { render } from "@testing-library/react";
import { MemoryRouter } from "react-router";
import Header from "./";

describe("Components", () => {
  describe("Header", () => {
    it("renders correctly", () => {
      const { container } = render(
        <MemoryRouter>
          <Header />
        </MemoryRouter>
      );

      expect(container.firstChild).toMatchSnapshot();
    });
  });
});
