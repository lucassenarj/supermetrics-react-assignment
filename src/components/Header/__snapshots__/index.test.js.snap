// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components Header renders correctly 1`] = `
<header
  class="header"
>
  <a
    href="/dashboard"
  >
    <img
      alt="Home"
      src="logo.png"
    />
  </a>
  <nav
    class="menu"
  >
    <li
      class="menu-item"
    >
      <a
        href="/dashboard"
      >
        Blog
      </a>
    </li>
    <li
      class="menu-item"
    >
      <a
        href="/about"
      >
        About
      </a>
    </li>
    <li
      class="menu-item"
    >
      <a
        href="/contact"
      >
        Contact
      </a>
    </li>
  </nav>
</header>
`;
